src/components/ButtonTag/ButtonTag.module.scss

```diff
@@ -1,8 +1,8 @@
 @use "src/styles/colors";
 
-$button-height: 3rem;
-$icon-size: 1.5rem;
-$label-size: 0.7rem;
+$button-height: 2rem;
+$icon-size: 0.8rem;
+$label-size: 0.5rem;
 
 $inner-shadow: inset 0 2px colors.getColor("tertiary", 60);
 
```

Comment:
Instead of having two classes have a lot of the same properties (buttonTagWrapper and buttonTagWrapperDisable) put all the common properties in buttonTagWrapper and make a separate class called `disable`